spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context 'factory tests' do
```

Comment:
Why this is context?
I'd say it is describe.

Reply:
`When describing a context, start its description with "when" or "with".`
http://www.betterspecs.org/#contexts